vector/src/main/res/xml/vector_settings_notifications.xml

```diff
@@ -43,9 +43,10 @@
 
     </im.vector.app.core.preference.VectorPreferenceCategory>
 
-    <im.vector.app.core.preference.VectorPreferenceCategory
+    <!-- Tchap : no email notifications -->
```

Comment:
```suggestion
    <!-- Tchap: no email notifications -->
```
Just remove the whitespace to have same template everywhere and to list all tchap comments easily.